beancount/ops/documents_test.py

```diff
@@ -42,7 +47,7 @@ def test_process_documents(self):
 
               2014-07-10 document Liabilities:US:Bank  "does-not-exist.pdf"
 
-            """).replace("ROOT", self.root)
+            """).replace("ROOT", self.root.replace("\\", r"\\"))
```

Comment:
Why modify a test that you skip anyway?

Reply:
make it easier to enable after we fix printer

Reply:
Which fix to the printer do you have in mind? How would changing the test in this way help?

If the change to the printer you have in mind it the one that would make it escape file paths, as we discussed in the other PR, the change here would actually produce wrong results, no more correct results: backslashes in the file path would end up being escaped twice. If you have something else in mind, please elaborate.

Reply:
no matter we fix printer or not the input is wrong on Windows

Reply:
I think you mis-read the code here, this is the input of parser, and backward slashes are expected to be escaped, right? `self.root` is the fileapth not being escaped.